go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/icinga/icingadb
 
 go 1.23.0
 
+toolchain go1.24.1
```

Comment:
Dunno why the toolchain was bumped to the latest version. Within go-sqlite3, the `go.mod` requires Go version 1.19, <https://github.com/mattn/go-sqlite3/blob/v1.14.25/go.mod>.

Locally, I have removed this line and re-ran `go mod tidy`. Doing so has not reintroduced this line.

Dunno, cosmic rays?